check-tokens/index.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const path = require('path')
+
+const keysToCheck = [
+	'borderWidth.focus',
```

Comment:
And BTW, I need to check this list one more time. I think I have some items missing 😕 😃 But that's on me... I'll fix it in this PR.